fastai2/data/core.py

```diff
@@ -86,10 +86,13 @@ def _pre_show_batch(self, b, max_n=9):
         if not is_listy(b): b,its = [b],L((o,) for o in its)
         return detuplify(b[:self.n_inp]),detuplify(b[self.n_inp:]),its
 
-    def show_batch(self, b=None, max_n=9, ctxs=None, show=True, **kwargs):
+    def show_batch(self, b=None, max_n=9, ctxs=None, show=True, unique=False, **kwargs):
+        old_get_idxs = self.get_idxs
+        self.get_idxs = (lambda: Inf.zeros) if unique else old_get_idxs
```

Comment:
Not need to make change if unique is false so I'd just do 
```
if unique: 
    old_get_idxs = self.get_idxs
    self.get_idxs = (lambda: Inf.zeros)
```
And test at the last line again.

Reply:
This would throw an error at line 95 though.

Can we instead do:
```python
old_get_idxs = self.get_idxs
if unique: self.get_idxs = lambda: Inf.zeros
```

Line 95 is required right? We need to make sure to revert any changes made to `self.get_idxs`

Reply:
(Sorry looks like GitHub held my comment since last night because it was requiring an extra click)

Reply:
Not a problem, I'll make the modifications =)